firmware/network/rpl_protocol/doc.txt

```diff
@@ -4,16 +4,16 @@
 
 ### What goes here?
 
-In this module you will find the functions to start rpl and add a node dodag (Raiz) 
+In this module you will find the functions to start rpl and add a node dodag (Raiz)
 or dag (Slave) first rpl is initialized then it is verified if the node is dodag or dag and
-they are started an ipv6 is added where the dodag will be hosted one Once the dodag is started, 
+they are started an ipv6 is added where the dodag will be hosted one Once the dodag is started,
```

Comment:
if they are started, an ipv6 address is added where the dodag will be hosted once the dodag is initialized,